ui/mirage/common.js

```diff
@@ -17,7 +17,7 @@ export const DATACENTERS = provide(
 );
 
 export const HOSTS = provide(100, () => {
-  const ip = Math.random() > 0.5 ? faker.internet.ip() : `[${ipv6()}]`;
+  const ip = Math.random() > 0.5 ? faker.internet.ip() : `[${faker.internet.ipv6()}]`;
```

Comment:
Hey @backspace 

Just saw this come in and wondered if these `Math.random`s are going to affect your work on stabilising the randomness of mirage/faker via a seed? Just an observation for now, but changing all these `Math.random`s to use fakers equivalent would probably be needed at some point (probably future PR)